.github/workflows/pypi-publish.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: setup python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: 3.11
```

Comment:
This can just be 3.12, we don't need to publish using the same version of python we're testing on.